site/docs/guides/admin/kubernetes/chart_lab.md

```diff
@@ -137,6 +137,17 @@ As before, you can define an Ingress, or use nodeports instead if preferred.
 
 Now go to https://localhost:8091/coco to access the React UI. Login as 'garygeeke',password 'admin'.
 
+## Accessing the Ploymer UI
+
+The same applies to the service exposing Ploymer UI via nginx
+
+```shell
+$ kubectl port-forward service/lab-nginx 8443:443
```

Comment:
I've never been a fan, I guess I like a server-side fix so I can share URL for review etc.

*BUT* I think you are spot-on here. For one simple way that works for remote + local k8s, and is basically independent of load balancers, complex ingress configurations etc, this will work just fine.

So Agree.. (any other docs around ingress etc is nice for the future, but unnecessary now)

Reply:
The only remaining piece might be to update the notebook where we refer to localhost - perhaps need to point back to this doc section (maybe via an anchor) rather than repeat the info there. either way is fine.